torch/_export/utils.py

```diff
@@ -211,6 +212,29 @@ def _getattr(model: torch.fx.GraphModule, attr_name: str):
     return params_buffers_to_node_meta
 
 
+def _maybe_find_pre_dispatch_tf_mode_for_export():
+    if not torch._C._is_torch_function_mode_enabled():
+        return None
+
+    torch_function_mode_stack = torch.overrides._get_current_function_mode_stack()
```

Comment:
There's a lot of these stack search utilities all around everywhere, maybe not this PR but any chance we can centralize them all in overrides? So it's easier to see all of them in one place.